### DISCUSSION TITLE
Under what circumstances will the value in the parmas of vue router be a string array?

### DISCUSSION THREAD

**Discussion by Bo-Teng** [vuejs/router#2289] [Help and Questions] [answered]
I saw in the documentation that the value of the RouteParams property may be an array, but I didn't find what would be the case.

**Comment by skirtles-code** [comment, marked as answer]
For repeatable params. See <https://router.vuejs.org/guide/essentials/route-matching-syntax.html#Repeatable-params>.

e.g.:

```js
const routes = [
  // matches /one, /one/two, /one/two/three, etc.
  { path: '/:chapters+' },
  // matches /, /one, /one/two, /one/two/three, etc.
  { path: '/:chapters*' },
]
```
